pkgs/development/libraries/cctz/default.nix

```diff
@@ -1,4 +1,6 @@
-{ lib, stdenv, fetchFromGitHub, darwin }:
+{ lib, stdenv, fetchFromGitHub, darwin
+, enableStatic ? stdenv.targetPlatform.isStatic
+}:
```

Comment:
```suggestion
{ lib, stdenv, fetchFromGitHub, Foundation }:
```

Reply:
`enableStatic` is used bellow.

Reply:
I edited your comment to directly use ``stdenv.targetPlatform.isStatic``

Reply:
I see. :-)